armi/reactor/converters/tests/test_blockConverter.py

```diff
@@ -18,18 +18,63 @@
 
 import numpy as np
 
+from armi.physics.neutronics.isotopicDepletion.isotopicDepletionInterface import (
+    isDepletable,
+)
+from armi.reactor import blocks, components, grids
 from armi.reactor.converters import blockConverters
-from armi.reactor import blocks
-from armi.reactor import components
 from armi.reactor.flags import Flags
 from armi.reactor.tests.test_blocks import loadTestBlock
-from armi.reactor.tests.test_reactors import loadTestReactor, TEST_ROOT
+from armi.reactor.tests.test_reactors import TEST_ROOT, loadTestReactor
 from armi.utils import hexagon
-from armi.reactor import grids
 from armi.utils.directoryChangers import TemporaryDirectoryChanger
-from armi.physics.neutronics.isotopicDepletion.isotopicDepletionInterface import (
-    isDepletable,
-)
+
+
+def buildSimpleFuelBlockNegativeArea():
+    """
+    Return a simple block containing fuel, clad, duct, and coolant.
+
+    The block has a negative-area gap between fuel and cladding for testing.
+    """
+    b = blocks.HexBlock("fuel", height=10.0)
+
+    fuelDims = {"Tinput": 25, "Thot": 600, "od": 0.76, "id": 0.00, "mult": 127.0}
+    cladDims = {"Tinput": 25, "Thot": 600, "od": 0.80, "id": 0.76, "mult": 127.0}
+    ductDims = {"Tinput": 25, "Thot": 600, "op": 16, "ip": 15.3, "mult": 1.0}
+    intercoolantDims = {
+        "Tinput": 400,
+        "Thot": 400,
+        "op": 17.0,
+        "ip": ductDims["op"],
+        "mult": 1.0,
+    }
+    coolDims = {"Tinput": 25.0, "Thot": 400}
+
+    fuel = components.Circle("fuel", "UZr", **fuelDims)
+    clad = components.Circle("clad", "HT9", **cladDims)
+    gapDims = {
+        "Tinput": 25,
+        "Thot": 600,
+        "od": "clad.id",
+        "id": "fuel.od",
+        "mult": 127.0,
+    }
+    gapDims["components"] = {"fuel": fuel, "clad": clad}
+    gap = components.Circle("gap", "Void", **gapDims)
+    duct = components.Hexagon("duct", "HT9", **ductDims)
+    coolant = components.DerivedShape("coolant", "Sodium", **coolDims)
+    intercoolant = components.Hexagon("intercoolant", "Sodium", **intercoolantDims)
+
+    b.add(fuel)
+    b.add(gap)
+    b.add(clad)
+    b.add(duct)
+    b.add(coolant)
+    b.add(intercoolant)
+
+    b.getVolumeFractions()  # TODO: remove, should be no-op when removed self.cached
```

Comment:
`TODO`?  The ARMI [Developer Manual](https://terrapower.github.io/armi/developer/standards_and_practices.html#general-do-s-and-don-ts) says we shouldn't add any more `TODO`s to the ARMI repo. If there is a bug or feature, we should open a ticket instead.

Reply:
this is copy/paste from another unit test, I just forgot to delete the comment